caffe2/operators/quantized/int8_resize_nearest_op.cc

```diff
@@ -12,6 +12,7 @@ OPERATOR_SCHEMA(Int8ResizeNearest)
     .Arg("Y_zero_point", "Output tensor quantization offset")
     .Arg("width_scale", "Scale along width dimension")
     .Arg("height_scale", "Scale along height dimension")
+    .Arg("output_size", "Output dimensions (HxW)")
```

Comment:
Let's mention when output_size and scale both exist, which one will be the source of the truth.